### ISSUE TITLE
VAE Pretraining and Incorporation into HL-Net

### ISSUE THREAD

**Issue by fridells51** [noahdrakes/mldl-final#1]
The VAE needs to be pre-trained on peaceful scenes.

(0.) The current implementation of the VAE was written with the intention that we are creating latent representations of the input then passing those values to the HL-net model. Instead, a pretrained VAE would act as a discriminatory component of the HL-net (because it was trained only on peaceful images, it will produce large reconstruction loss for a violent image)

1. Find the inputs labeled peaceful, and train the VAE using the author's implementation
2. The pretrained model needs to be added as an arm to the HL-Net. The pretrained VAE will be evaluated on input data, and the reconstruction loss will be generated with an evaluation call.

**Comment by fridells51** [comment]
see #2 very related issue

**Comment by fridells51** [comment]
Got the VAE training. See functional model in the latest commits/push to branch `gus-edits`
